eng/Analyzers.props

```diff
@@ -6,7 +6,7 @@
     <EnableAnalyzers Condition="'$(DotNetBuildFromSource)' == 'true'">false</EnableAnalyzers>
   </PropertyGroup>
   <ItemGroup Condition="'$(EnableAnalyzers)' == 'true'">
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0-preview3.21153.3" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0-preview1.21554.2" PrivateAssets="all" />
```

Comment:
should it be dependency? or should we change to stable 6.0 version?

Reply:
Opened seperate issue for it: https://github.com/dotnet/templating/issues/4114